app/assets/javascripts/navbar.js

```diff
@@ -0,0 +1,13 @@
+$( document ).on('turbolinks:load', function() {
+    let navbarTop = $(".main-navbar");
+    $(function() {
+        $(window).scroll(function () {

```

Comment:
give meaningful names to javascript methods in order to describe what you've done inside it